src/pydocstyle.py

```diff
@@ -1592,9 +1592,11 @@ def check_unicode_docstring(self, definition, docstring):
 
         # Just check that docstring is unicode, check_triple_double_quotes
         # ensures the correct quotes.
-        if docstring and sys.version_info[0] <= 2:
-            if not is_ascii(docstring) and not docstring.startswith(
-                    ('u', 'ur')):
+        if docstring and not docstring.startswith('u'):
+            if sys.version_info < (3, 0) and not is_ascii(docstring):
```

Comment:
Since this would add the first runtime dependency to the project, and thus break it's advertised self-containedness, I suggest you integrate the library as you best see fit. I will then rebase / adjust the PR accordingly. However, I think the benefit at this particular location would by marginally, as six does not offer minor version (range) distinction.